components/script/dom/range.rs

```diff
@@ -286,38 +292,38 @@ impl Range {
             .iter()
             .for_each(|s| s.queue_selectionchange_task());
     }
-}
 
-impl RangeMethods for Range {
-    // https://dom.spec.whatwg.org/#dom-range-startcontainer
-    fn StartContainer(&self) -> DomRoot<Node> {
-        self.start.node.get()
+    fn abstract_range(&self) -> &AbstractRange {
+        self.upcast::<AbstractRange>()
```

Comment:
Couldn't you just replace calls to `self.abstract_range()` with `self.abstract_range` and refer to the struct member directly? (after the rename)?